recipe/meta.yaml

```diff
@@ -1,12 +1,10 @@
-{% set version = "6.0.0rc5" %}
-
 package:
   name: gmt
-  version: {{ version }}
+  version: 6.2.0dev
```

Comment:
@seisman, do you think it will be a better idea to put the date in the version (e.g. `6.2.0dev20200729`)? Or we could have a shortened git revision (e.g. `6.2.0dev9d2cbb0`). Maybe add a hyphen in there or something too.

It's actually a bit hard bumping the build number automatically with Github Actions, and after some thought, I think the build number should only change with if the GMT build dependencies change (rather than a change in the GMT package itself).

Reply:
> It's actually a bit hard bumping the build number automatically with Github Actions, and after some thought, I think the build number should only change with if the GMT build dependencies change (rather than a change in the GMT package itself).

Yes, we should only bump the build number when we want to rebuilt the same tarball, due to dependencies changes or packaging errors.

> @seisman, do you think it will be a better idea to put the date in the version (e.g. `6.2.0dev20200729`)? Or we could have a shortened git revision (e.g. `6.2.0dev9d2cbb0`). Maybe add a hyphen in there or something too. Will need to look at https://semver.org/ for guidance.

GMT itself reports a version string `6.2.0_9d2cbb0_2020.07.28` for development branches. We should also add something to the conda package version strings. Better to check what other projects do.

Reply:
> GMT itself reports a version string `6.2.0_9d2cbb0_2020.07.28` for development branches. We should also add something to the conda package version strings. Better to check what other projects do.

Yes I'd prefer a version sting like that with both the shortened git revision and date. Been looking at https://stackoverflow.com/questions/8055029/is-there-a-simple-way-to-git-describe-a-remote-repository and https://stackoverflow.com/questions/39120781/how-to-get-git-describe-like-output-from-a-remote-repository to see how to tease that out without doing a shallow clone of the GMT repository.

Reply:
Actually, a shallow clone might be needed since we can't just pull the unreleased '6.2.0' using `git ls-remote`. Or we could just hard code the '6.2.0' part (since it will only change half a year or so) and bump it manually when needed.

> Better to check what other projects do.

It doesn't look consistent when scanning through the other conda-forge packages with the label 'dev' at https://anaconda.org/conda-forge/repo?label=dev. Most of them use the 'dev' branch for rc releases so it's just 'rc1', 'rc2', etc.